dcos/subcommand.py

```diff
@@ -1,12 +1,17 @@
 from __future__ import print_function
 
+import hashlib
 import json
 import os
+import platform
 import shutil
+import stat
 import subprocess
 import sys
+import zipfile
 from subprocess import PIPE, Popen
 
+import dcos.http
```

Comment:
It's actually causing other import issues. Filed #602, and will use `requests` until we fix.